tests/storage/outofprocess_test.py

```diff
@@ -49,6 +51,23 @@ def oop_cleanup():
     oop.stop()
 
 
+@pytest.fixture(
+    scope="function",
```

Comment:
function is the default, why do we need to specify it? No other fixture use scope="function". If we don't have a good reason to change all fixture, we better be consistent with existing code.